core/src/components/content/test/standalone/content.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('content: standalone', () => {
+  test('should not have visual regressions', async ({ page }) => {
```

Comment:
Do we need the standalone tests?

Reply:
The standalone tests were kept since `ion-content` references `ion-app` when calculating the size. I noted this under Other Info in the PR description, but I'll take this as a nudge to put info like that somewhere more visible 🤭

Reply:
Oh right, thanks!